EpiResponsivePicture/TypeScript/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "epi-responsive-picture",
+  "version": "1.0.0",
+  "description": "",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsup",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Forte Digital",
+  "license": "ISC",
+  "devDependencies": {
+    "ts-node": "^10.9.2",
+    "tsup": "^8.0.1",
+    "typescript": "^5.3.3"
+  },
+  "dependencies": {
+    "@types/react": "^18.2.47",
+    "react": "^18.0.0"
```

Comment:
I wonder if the 18 version is a must? I can imagine projects with older versions of React wanting to use this package. Wouldn't this entry force them to upgrade major version of React? Do we really require React v18?

Reply:
@tomaszmadeyski what should be you suggested lowest supported version then?

Reply:
this definitely not a must, the reason for having 18 here is that first consumer of this packaged has react 18

Reply:
I think 17 as it's the newest version of React supported by React.NET